### DISCUSSION TITLE
API url

### DISCUSSION THREAD

**Discussion by tr11** [vectordotdev/vector#19592] [Q&A]
Is there a way to configure the API url, say to serve the graphql endpoint under a subpath?

**Comment by jszwedko** [comment]
Hi @tr11 !

There isn't currently a way to change the path where the GraphQL endpoint is hosted. It's hard coded to `/graphql`. I'm curious what your need is for adjusting it to understand the best way to address it.

**Reply by tr11** [reply]
Thanks for the quick reply.  We run all our telemetry and monitoring tools under specific subpaths behind a reverse proxy. It's easy enough to do a url rewrite at the reverse proxy level, which we then pair with a custom graphql playground with the correct path. Just wondering whether that feature existed so we could use the included playground defaulting  to the correct url.
